src/server.js

```diff
@@ -17,12 +19,27 @@ app.use(cors());
 
 app.use(morgan("combined"));
 
+// Used by certain endpoints for verification
+async function verify(token, email) {
+  const ticket = await client.verifyIdToken({
+    idToken: token,
+    audience: process.env.REACT_APP_GOOGLE_CLIENT_ID
+  });
+  const payload = ticket.getPayload();
+  return payload.email == email;
+}
+
 // Test API method 
 app.get("/test", (req, res) => {
   res.status(200).send("Hello World!");
 });
 
 // Add API methods here
+app.post("/users", (req, res) => {
```

Comment:
?? doesn't do anything?

Reply:
You see I messed up doing these...so sorry about it. These will be fully implemented in another pull request when I work on the back end.

Reply:
file bug and link